CONTRIBUTING.md

```diff
@@ -1,36 +1,210 @@
 # Contributing to Vireo
 
-## Getting Started
-To begin development on Vireo, make sure to start by [forking the repository](https://help.github.com/fork-a-repo/).
+This guide will walkthrough the different workflows recommended for developing Vireo.
```

Comment:
walkthrough with no space is a noun/adjective. Use "walk through" instead. (http://www.dictionary.com/browse/walkthrough)